source/collections/LinkedList.ooc

```diff
@@ -0,0 +1,164 @@
+import structs/List
+
+LinkedList: class <T> {
+	_size = 0 : SizeT
+	size ::= this _size as Int
+	_head: Node<T>
+	head ::= this _head
+	init: func {
+		this _head = Node<T> new()
+		this _head prev = this _head
+		this _head next = this _head
```

Comment:
Okay, I can see the point of doing things like this, reading the other functions.